NMS-14619: Quick Start: Determine service availability (#5350)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc

  <downtime interval="600000" begin="43200000" end="432000000" />
</package>
----

> <1> Polling interval in the packages is 300 seconds and 30 seconds.

Reviewer: ```suggestion
<1> Polling interval for the packages is 300 seconds and 30 seconds.
```

> <2> Different polling interval for the ICMP service.
> <3> Different retry settings for the ICMP service.
> <4> Different timeout settings for the ICMP service.

Reviewer: ```suggestion
<2> Custom polling interval for the ICMP service.
<3> Custom retry settings for the ICMP service.
<4> Custom timeout settings for the ICMP service.
```

The last polling package on the service will be applied.

> You can use this to define a less specific catch-all filter for a default configuration.

Reviewer: ```suggestion
You can use this to define a less specific catch-all filter as the default configuration.
```

> Use a more specific polling package to overwrite the default setting.

Reviewer: ```suggestion
Use a more specific polling package to overwrite the default settings.
```

> In the above example, all IP interfaces in `192.168.1/24` or `2600:/64` will be monitored with ICMP with different polling, retry, and timeout settings.

Reviewer: ```suggestion
In the previous example, all IP interfaces in `192.168.1/24` or `2600:/64` will be monitored with ICMP using custom polling, retry, and timeout settings.
```

The web UI displays which polling packages are applied to the IP interface and service.

> The IP interface and service pages show which polling package and service configuration are applied for this specific service.

Reviewer: The second line here seems redundant. I'd suggest removing it or combining the two.

> .Polling package applied to IP interface and service

Reviewer: ```suggestion
.Polling package settings applied to IP interface and service
```

> In the above example, all IP interfaces in 192.168.1/24 or 2600:/64 will be monitored with ICMP with different polling, retry, and timeout settings.

Reviewer: ```suggestion
In the above example, all IP interfaces in `192.168.1/24` or `2600:/64` will be monitored with ICMP with different polling, retry, and timeout settings.
```
